_vector-search/ai-search/hybrid-search/collapse.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: default
+title: Using collapse with a hybrid query
+parent: Hybrid search
+grand_parent: AI search
+has_children: false
+nav_order: 10
+---
+
+# Using collapse with hybrid query
+**Introduced 3.1**
+{: .label .label-purple }
+
+You can collapse on a field by specifying the `collapse` parameter in the search query.
+This will return the highest scoring document for each unique value of that field, up to the specified size in the query.
+The collapse parameter requires the field being collapsed to be of either a `keyword` or a `numeric` type.
+
+Note: Inner hits compatibility within the collapse description is not currently supported for hybrid query.
```

Comment:
I think we need to add more notes, not sure how to organize them though:
- we don't support collapse via processor (I think we do not), maybe more details here: will there be an error, or results will be off etc.
- performance impact may be higher with large result sets
- aggregations run on pre-collapsed results 
- collapse affects pagination and total hit counts

Reply:
Let's add middle two and the pagination.
As for the processor, I think results can be different from one that we can see when we pass a parameter, due to processor not seeing all search hits from shards. It's same for other queries, but I think it's worth mentioning.